scripts/buildkite/tpp-gpu.yml

```diff
@@ -8,19 +8,29 @@ env:
   NPROCS_LIMIT_LINK: "8"
   #LIBXSMM_VERBOSE: 4
   #LIBXSMMFETCH: 1
-  TPP_CLANGVER: "16.0.4"
-  TPP_GCCVER: "9.5.0" # Use an older version to match CUDA requirements
-  TPP_CUDAVER: "11.5"
-  # ASAN flags required due to incompatibility with CUDA
-  # See: https://github.com/google/sanitizers/issues/629
-  ASAN_OPTIONS: "protect_shadow_gap=0:replace_intrin=0:detect_leaks=0:${ASAN_OPTIONS}"
 
 steps:
   - label: "LLVM"
-    command: "${BUILDKITE_BUILD_CHECKOUT_PATH}/scripts/buildkite/check_llvm.sh"
+    command: "scripts/buildkite/check_llvm.sh"
   - wait
 
-  - label: "TPP-MLIR-clang"
-    command: "${SRUN} --partition=nv-v100 --time=0:30:00 -- \
+  - label: "TPP-MLIR-nvidia"
+    command: "${SRUN} --partition=nv-v100,nv-a100 --time=0:30:00 -- \
               'KIND=Debug COMPILER=clang LINKER=lld GPU=1 SANITIZERS=1 CHECK=1 \
-              $${BUILDKITE_BUILD_CHECKOUT_PATH}/scripts/buildkite/build_tpp.sh'"
+              scripts/buildkite/build_tpp.sh'"
+    env:
+      # ASAN flags required due to incompatibility with CUDA
+      # See: https://github.com/google/sanitizers/issues/629
+      ASAN_OPTIONS: "protect_shadow_gap=0:replace_intrin=0:detect_leaks=0:${ASAN_OPTIONS}"
+      TPP_CLANGVER: "16.0.4"
+      TPP_GCCVER: "9.5.0" # Use an older version to match CUDA requirements
+      TPP_CUDAVER: "11.5"
+
+  - label: "TPP-MLIR-intel"
```

Comment:
Why don't we have forced tools for Intel GPU?

Reply:
Not yet. We can add later. ATM, nothing is actually executed on Intel GPU, and further development (incl. source level) is needed. For example, we may need IMEX to be fiddled into our LLVM, etc. I can also enforce some tools if your prefer.